packages/flutter_tools/bin/macos_build_flutter_assets.sh

```diff
@@ -33,18 +31,6 @@ if [[ -n "$FLUTTER_TARGET" ]]; then
     target_path="${FLUTTER_TARGET}"
 fi
 
-# Set the track widget creation flag.
-track_widget_creation_flag=""
-if [[ -n "$TRACK_WIDGET_CREATION" ]]; then
-  track_widget_creation_flag="--track-widget-creation"
-fi
```

Comment:
Removing this regressed the bug where hot reload crashes in IDEs; I'm not sure why it was taken out.

I'll send a follow-up PR shortly.